app/assets/images/payment_icons/nelo.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-nelo">
```

Comment:
Can we put the code in 1 line just like the rest of brand icons?

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

Reply:
Absolutely! Just updated the PR to address this.